server/server.py

```diff
@@ -35,6 +35,14 @@ def check():
     return "🚀"
 
 
+@app.route('/slack/commands', methods=['POST'])
```

Comment:
Can anything else be removed here (i.e. the Slack events adapter)? I'm not actually sure, and if it's risky we can just keep it for now and remove it in another PR.